content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -70,6 +70,39 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 
 {{< /note >}}
 
+### Enable dual-stack via kubeadm
+
+Install kubeadm following the steps from the [Installing Kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) documentation.
+
+Make sure that nodes allow IPv6 forwarding, if not, run `sudo sysctl -w net.ipv6.conf.all.forwarding=1` on every node in the cluster.
+
+* Notice that `kubeadm upgrade` will change IPv6DualStack to true by default if feature gate is not set in old cluster. However, cluster-cidr and service-cidr modification are not supported.
+
+#### Create a dual-stack cluster
+
+You must pass your `podCidr` and `serviceCidr` value to `kubeadm init`, like so:
+
+```
+kubeadm init --feature-gates IPv6DualStack=true --pod-network-cidr=172.30.0.0/16,fefe:ffff:0::/48 --service-cidr=172.31.0.0/16,fefe:ffff:1::/108
+```
+
+Currently, `kube-controller-manager` flags `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` are setting with the default value. See [enable IPv4/IPv6 dual stack](/docs/concepts/services-networking/dual-stack#enable-ipv4ipv6-dual-stack).
+
+There is a limitation here, `--apiserver-advertise-address` flag doesn't support dual-stack.
```

Comment:
we should first mention the config fields:
InitConfiguration -> localAPIEndpoint
and 
JoinConfiguration -> controlPlane -> localAPIEndpoint

https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2

we can also mention the flag as they are GA, it's just that we recommend the config to users since it's versioned.

Reply:
Done